birdhouse/read-configs.include.sh

```diff
@@ -88,29 +88,38 @@ read_env_local() {
 source_conf_files() {
   dirs=$1
   conf_locations=$2
+  _old_adir=$adir # a posix-compliant way to make this a local variable'
```

Comment:
`adir` is a very non specific var.  If it is used in another function before calling this one, would it mess this one up?

Reply:
It shouldn't matter... but this has been changed anyway with the new stack implimentation